deltametrics/sample_data/cube.py

```diff
@@ -18,19 +23,32 @@
 ------------------------
 
 :meth:`tdb12` : `ndarray`
-  This data cube is from Tulane Delta Basin expt 12. 
+    This data cube is from Tulane Delta Basin expt 12.
 
-:meth:`rcm1` : `ndarray`
-  This data cube is from the DeltaRCM model. 
+:meth:`rcm8` : `ndarray`
+    This data cube is from the pyDeltaRCM model.
 
 """
 
+import sys
+import os
 
 import numpy as np
+import netCDF4
+
+from ..cube import Cube
+from ..io import NetCDFIO, HDFIO
+
 
 def tdb12():
-  return np.array([[]])
+    raise NotImplementedError
```

Comment:
The whole thing would be kind of a pain since just the topo is about 2 GB, but we could provide a small T x X x Y square cube.

Reply:
yeah that's fine. Not worried about it for now. Just put there as kind of a placeholder/reminder.

Reply:
I'll send something to you and Jay to play with.